keyboards/fungo/readme.md

```diff
@@ -0,0 +1,24 @@
+# fungo split keyboard (fungo)
+
+
+![full](https://user-images.githubusercontent.com/107382014/176942337-31313d0e-1588-475b-877b-55e0a74f06e6.jpg)
+
+A split keyboard for my hands.
+
+Keyboard Maintainer: [bojiguard](https://github.com/bojiguard/)  
+Hardware Supported: Pro Micro, elite-c, nicenano  
+Hardware Availability: [fungo split keyboard](https://github.com/bojiguard/fungo-split-keyboard)  
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
add bootloader information